server/server.go

```diff
@@ -672,6 +676,19 @@ func (s *Server) Serve(ctx context.Context) {
 		provide(s.auth0OAuth(logger, auth)),
 	}
 
+	var basicAuthenticator *basicAuth.BasicAuth
+	if len(s.BasicAuthHtpasswd) > 0 {
+		logger.
+			WithField("component", "server").
+			WithField("realm", s.BasicAuthRealm).
+			WithField("htpasswd", s.BasicAuthHtpasswd).
```

Comment:
Will this leak the password in the logs?

Reply:
No, it prints out the location of the .htpasswd file.